src/Cardano/WalletLayer/Wallet.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE TypeOperators #-}
+
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- | Module provides internal implementation of
+-- | wallet api
+
+module Cardano.WalletLayer.Wallet
```

Comment:
Let's have that in the same module as `WalletLayer` until we see a good reason to move them out :)

Reply:
done